euphonic/util.py

```diff
@@ -723,7 +765,8 @@ def _get_supercell_relative_idx(cell_origins: np.ndarray,
             if np.all(dist_min <= 16*sys.float_info.epsilon):
                 break
         if np.any(dist_min > 16*sys.float_info.epsilon):
```

Comment:
Why 16?

Reply:
Don't know, don't dare change it without deeper investigation.

Magic numbers should be CONSTANTS but they also need good names.